modules/raw-rsa-keyring-browser/src/raw_rsa_keyring_web_crypto.ts

```diff
@@ -203,19 +204,22 @@ export class RawRsaKeyringWebCrypto extends KeyringWebCrypto {
     const backend = await getWebCryptoBackend()
 
     if (isFullSupportWebCryptoBackend(backend)) {
+      // @ts-ignore // TS2769: No overload matches this call
```

Comment:
If we can't convince TS the types work out, the comment should at least explain why 'No overload matches this call' is not real and what really is going on :)

Reply:
I tried the idea discussed on our call. No luck.
I tried ideas around that. Defining a type, using {}, etc.

The only idea I did not try was explicitly creating a type that was either Json or the other override.
That might work. 
But I think I am putting this down as I move off call... not sure yet.